src/core/load-fragment.c

```diff
@@ -5199,6 +5199,38 @@ int config_parse_temporary_filesystems(
         }
 }
 
+int config_parse_private_tmp(
+                const char* unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        ExecContext *c = ASSERT_PTR(data);
+        bool fatal = ltype;
```

Comment:
you are not actually setting ltype to anything but zero? seems overkill to bother with "fatal" here..

Reply:
copy copypasted from another implementation above tbh